src/components/molecules/AuthForm/index.tsx

```diff
@@ -24,14 +24,38 @@ const AuthForm = ({ isLogInForm = false }: AuthFormProps) => {
         >
```

Comment:
I was thinking about it while creating this component and now you confirmed that I was right- it should be 2 separate components. But now I'm wondering about authSchema and useAuthForm: should every form has its own or can it stay as it is (one schema and custom hook for both forms)? If it can stay as it is, in which folder should I store authSchema and useAuthForm?